src/ui/components/Collections/styles.jsx

```diff
@@ -63,6 +63,9 @@ export default makeStyles(
         backgroundColor: theme.palette.common.grayHighlight
       }
     },
+    tableRowEdit: {
+      backgroundColor: '#e6f7f5'
```

Comment:
I'm not sure whether these two new colors will be reused but consider adding them to the palette